src/app/docs/selection-box/selection-box-docs.component.html

```diff
@@ -0,0 +1,256 @@
+        >
+        <p>
+            On smaller viewports, selection boxes hide their icons and use smaller typography and spacing.
+        </p>
+
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/selection-box/selection-box-behavior-1-modern.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+        
+      <h3>States</h3>
+        <table class="sky-docs-table">
+          <tr>
+            <td class="sky-docs-table-cell sky-docs-table-cell-primary">
+              Unselected
+            </td>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+                imageSource="~/assets/img/guidelines/selection-box/selection-box-states-base-modern.png"
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+          </tr>
+          <tr>
+            <td class="sky-docs-table-cell">
+              Hover
+            </td>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+                imageSource="~/assets/img/guidelines/selection-box/selection-box-states-hover-modern.png"
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+          </tr>
+          <tr>
+            <td class="sky-docs-table-cell">
+              Focus
+            </td>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+                imageSource="~/assets/img/guidelines/selection-box/selection-box-states-focus-modern.png"
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+          </tr>
+          <tr>
+            <td class="sky-docs-table-cell">
+              Selected
+            </td>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+                imageSource="~/assets/img/guidelines/selection-box/selection-box-states-selected-modern.png"
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+          </tr>
+          <tr>
+            <td class="sky-docs-table-cell">
+              Disabled
+            </td>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+                imageSource="~/assets/img/guidelines/selection-box/selection-box-states-disabled-modern.png"
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+          </tr>
+        </table>
+        
+    </sky-docs-demo-page-section>
+      
+    <sky-docs-demo-page-section
+      heading="Content"
+    >
+      
+      <div class="sky-text-block">
+        <p>
+            Keep titles and descriptions short and concise.  Limit descriptions to a single sentence.
+        </p>
+      </div>
+      
+    </sky-docs-demo-page-section>
+      
+    <sky-docs-demo-page-section
```

Comment:
Do we need to show how to arrange these alongside other content on a page, or is that a can of worms we want to leave closed?

Reply:
I think it's a can of worms, _unless_ we consider "Transactional Page" as a candidate for a shiny new page template.